blog_post_collaboration/kedro_ibis.md

```diff
@@ -0,0 +1,263 @@
+
+In practice, a machine learning (software) engineer often ends up rewriting proof-of-concept code to be more performant and scalable, but this presents new challenges. First, it takes time to reimplement everything properly in another technology, and developers frequently underestimate the effort required. Also, rewritten logic, intentionally or not, doesn't always correspond 1:1 to the original, leading to unexpected results (read: bugs). Proper testing during development can mitigate, but not eliminate, correctness issues, but expecting all data science code to be unit tested is wishful thinking. Last but not least, business stakeholders don't always understand why productionising code takes so much time and resources—after all, it worked before! Who knows? Maybe scaling data engineering code shouldn't be so hard...
+
+Given the ubiquity of pandas during the development phase, some execution engines started supporting pandas-compatible or similar Python dataframe APIs, such as the [pandas API on Spark](https://spark.apache.org/docs/latest/api/python/user_guide/pandas_on_spark/index.html) and [BigQuery DataFrames](https://cloud.google.com/bigquery/docs/reference/bigquery-dataframes). Case studies, like [this one from Virgin Hyperloop
+One](https://www.databricks.com/blog/2019/08/22/guest-blog-how-virgin-hyperloop-one-reduced-processing-time-from-hours-to-minutes-with-koalas.html), demonstrate how these solutions can accelerate the scaling process for data-intensive code. That said, building and maintaining a pandas-compatible API requires significant commitment from the engine developer, so no panacea exists.
+
+## The SQL solution
+
+Thankfully, there exists a standardized programming language that every database (and more-or-less every major computation framework) supports: SQL![^1]
+
+_If you like SQL,_ [dbt](https://www.getdbt.com) provides a battle-tested framework for defining and deploying data transformation workflows. [As of February 2022, over 9,000 companies were using dbt in production.](https://www.getdbt.com/blog/next-layer-of-the-modern-data-stack) Here on the Kedro team, we love dbt! In fact, we frequently recommend it for ELT (Extract, Load, Transform) workflows. Kedro and dbt share similar self-stated goals: [Kedro empowers data teams to write reproducible, maintainable, and modular Python code,](https://docs.kedro.org/en/stable/introduction/index.html#introduction-to-kedro) and ["dbt is a SQL-first transformation workflow that lets teams quickly and collaboratively deploy analytics code following software engineering best practices."](https://www.getdbt.com/product/what-is-dbt)
+
+> _"When I learned about Kedro (while at dbt Labs), I commented that it was like dbt if it were created by Python data scientists instead of SQL data analysts (including both being created out of consulting companies)."_
+>
+> —Cody Peterson, Technical Product Manager
```

Comment:
might want to add " @ Voltron Data" just to be clear I don't work at dbt Labs anymore